### DISCUSSION TITLE
Display last documents opened

### DISCUSSION THREAD

**Discussion by WAAH1** [sumatrapdfreader/sumatrapdf#4263] [Q&A]
Hello,

i would like to know if it's possible to display the historic (the last opened documents) instead of the most read documents as it is now

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/110057376/9ce619ed-33e4-44cb-a957-1ed3d5cca9c6)

many thanks for your help

**Comment by GitHubRulesOK** [comment]
"Frequently" is based on MOST opened order
The "File OPEN" order is the last read even if closed 
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/0fe2ae6b-e758-41f8-a175-619330ea11f9)

Once you close a session and reopen the order is the last CLOSED And thus now OPENED
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/5a96a14a-7a74-4d89-b17d-8e30cd2faf39)

**Reply by WAAH1** [reply]
Yeah you right, thanks for your help
But is there no possibility to display these last opened file directly on the home page ? instead of the most frequent open doc ?

Thanks

**Reply by GitHubRulesOK** [reply]
the number of thumbnails from opened files in prerelease is now 20 ?
 and if no "frequently read is taking first slot they would reflect simple order of opening, however if opened frequently they move to the top left in rank. However in [PRE_RELEASE](https://www.sumatrapdfreader.org/prerelease) home screen you can invoke CTRL K which gives a much larger history most recent at top
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/2682310f-6def-4355-9224-f471014b22a2)
.

**Reply by WAAH1** [reply]
wonderful, thanks ! 
i didn't know this shortcut

**Reply by dominiosc** [reply]
I also think it would be interesting to show the latest files on the home page. The problem is that we keep many files open permanently in the session, so we hardly see any files from the history.

An internal tab could be created to choose what to view, either the latest files or the most viewed files.

**Reply by GitHubRulesOK** [reply]
@dominiosc 

Hotkeys work on home page so to see recent in order **assign a key** for example # will open file history
```
Shortcuts [
	[
		Cmd = CmdCommandPalette #
		Key = #
		Name = File History
		ToolbarText = #
]
```
<img width="738" height="290" alt="image" src="https://github.com/user-attachments/assets/0b6b95e3-e3b8-4b62-b959-fe8f359901a4" />

**Reply by dominiosc** [reply]
Advanced users can certainly use commands, but desktop users only use the home tab for ease of use and because it's visually simpler.

**Comment by dominiosc** [comment]
In last update of pre-release the home tab show near 600 recently opened files.
Is possible set a number to show, for example, in my screen the best are 45 max.

**Reply by kjk** [reply]
is there a performance problem or you just don't like so many?

**Reply by dominiosc** [reply]
Not performance problem, the program open fast with last fix for opened files.
But I think they're excessive. If you don't consider limiting by parameters,
perhaps an option to click on "see more"... after the first page.
I don't see the need to scroll to see files I opened a long time ago, and I open many daily.
Thanks

**Reply by dominiosc** [reply]
If not by number, then by date, on shared computers, seeing so many open files on the home screen can become problematic. We've gone from 10 files to 600; the best solution would be to configure it somehow so that each user can adjust it to their needs.
